src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/configuration/BrregForwardingFilter.kt

```diff
@@ -0,0 +1,41 @@
+package no.brreg.informasjonsforvaltning.organizationcatalogue.configuration
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import javax.servlet.Filter
+import javax.servlet.FilterChain
+import javax.servlet.ServletRequest
+import javax.servlet.ServletResponse
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+private val LOGGER = LoggerFactory.getLogger(BrregForwardingFilter::class.java)
+
+@Component
+class BrregForwardingFilter(private val appProperties: AppProperties) : Filter {
+
+    override fun doFilter(request: ServletRequest?, response: ServletResponse?, chain: FilterChain?) {
+        if (request != null && response != null && chain != null) {
+            try {
+                val httpRequest = request as HttpServletRequest
+                val httpResponse = response as HttpServletResponse
+
+                val acceptContainsHtml: Boolean = httpRequest.getHeader("Accept")
+                    ?.contains("text/html")
+                    ?: false
+
+                if (acceptContainsHtml && httpRequest.servletPath.contains("organizations")) {
+                    val orgId = httpRequest.servletPath.substringAfter("organizations/", "")
+                    httpResponse.setHeader("Location", "${appProperties.enhetsregisteretUrl}$orgId")
+                    httpResponse.sendRedirect("${appProperties.enhetsregisteretUrl}$orgId")
```

Comment:
For å sende 303, må du visstnok gjere følgande:
```
You need to set the response status and the Location header manually.

response.setStatus(HttpServletResponse.SC_MOVED_PERMANENTLY);
response.setHeader("Location", "http://somewhere/");
```
Ref https://stackoverflow.com/questions/9034149/httpservletresponse-sendredirect-permanent

Reply:
var det ikke 303 du ville ha? `Moved permanently` er vel 301?